src/sage/quadratic_forms/quadratic_form__local_field_invariants.py

```diff
@@ -286,22 +285,16 @@ def _rational_diagonal_form_and_transformation(self):
 
 
 def signature_vector(self):
-    """
-    Returns the triple `(p, n, z)` of integers where
-
-    - `p` = number of positive eigenvalues
-    - `n` = number of negative eigenvalues
-    - `z` = number of zero eigenvalues
-
-    for the symmetric matrix associated to Q.
-
-    INPUT:
+    r"""
+    Return the triple `(p, n, z)` of integers where
 
-    (none)
+    - ``p`` = number of positive eigenvalues
+    - ``n`` = number of negative eigenvalues
+    - ``z`` = number of zero eigenvalues
 
```

Comment:
These are not input arguments. Hence they should be single-quoted.

Reply:
Should I make this type of change also here:
```rst
    OUTPUT: either ``D`` (if ``return_matrix`` is false) or ``(D, T)``
    (if ``return_matrix`` is true) where

    - ``D`` -- the diagonalized form of this quadratic form

    - ``T`` -- transformation matrix. This is such that
      ``T.transpose() * self.matrix() * T`` gives ``D.matrix()``.

    Both ``D`` and ``T`` are defined over the fraction field of the
    base ring of the given form.
```

Reply:
Yes.